### PR TITLE
aa: Add to repo 

### PR DESCRIPTION
**Pull request by mPaella**
copying [aa sdk repo](https://github.com/Crossmint/Wallet-SDK) into this repo

## Test plan

built package, then `npm link`ed it to the scw_demo, tested with the demo

https://github.com/Crossmint/crossmint-sdk/assets/93682696/a9c329d3-23cf-4a3a-ab2c-b713c38dd379

### CODE REVIEW

File: packages/client/wallets/aa/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (alfonso-paella) [inline packages/client/wallets/aa/LICENSE:1]: Hmm should we actually use this license... I'd remove it if in doubt

File: packages/client/wallets/aa/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@crossmint/client-sdk-aa",
+    "version": "0.0.9-alpha.6",
+    "author": "Paella Labs Inc",
```
Comment (alfonso-paella) [inline packages/client/wallets/aa/package.json:4]: Paella inc

File: packages/client/wallets/aa/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@crossmint/client-sdk-aa",
+    "version": "0.0.9-alpha.6",
```
Comment (alfonso-paella) [inline packages/client/wallets/aa/package.json:3]: Is this version seen externally? can we keep it in sync w other packages?

File: packages/client/wallets/aa/package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@crossmint/client-sdk-aa",
```
[the remark this answers is not in the document]
Comment (mPaella) [reply packages/client/wallets/aa/package.json:2]: yeah same as existing package
https://www.npmjs.com/package/@crossmint/client-sdk-aa

File: packages/client/wallets/aa/package.json (hunk 3 above)
Comment (mPaella) [reply packages/client/wallets/aa/package.json:3]: in sync, other repo should be archived after this
https://www.npmjs.com/package/@crossmint/client-sdk-aa

File: packages/client/wallets/aa/package.json (hunk 2 above)
Comment (mPaella) [reply packages/client/wallets/aa/package.json:4]: do u want me to change all the packages we have? they all use this same author

File: packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts [hunk 5]
```diff
@@ -0,0 +1,87 @@
+export const EVMBlockchain = {
```
Comment (alfonso-paella) [inline packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts:1]: This class should be in the core package, imported by wallets. but the team can do this on another PR

File: packages/client/wallets/aa/src/utils/constants.ts [hunk 6]
```diff
@@ -0,0 +1,3 @@
+export const ZERO_PROJECT_ID = "7016355c-c7b3-4d57-a50e-84e9d02d9c52";
```
Comment (alfonso-paella) [inline packages/client/wallets/aa/src/utils/constants.ts:1]: Hmmm not sure if this is something we are ok checking in in the code

File: packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts (hunk 5 above)
Comment (mPaella) [reply packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts:1]: yeah definitely, we should actually have a package like `@crossmint/common-sdk-base/core` wdyt?

Comment (alfonso-paella) [reply packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts:1]: sg

File: .npmrc [hunk 7]
```diff
@@ -0,0 +1,2 @@
+engine-strict=true
```
Comment (alfonso-paella) [inline .npmrc:1]: intended?

Comment (mPaella) [reply .npmrc:1]: yeah its intended, have the same thing in xmint main

File: packages/client/wallets/aa/package.json (hunk 2 above)
Comment (alfonso-paella) [reply packages/client/wallets/aa/package.json:4]: Mind filing a task and we do it later? But yes, this is the wrong company XDDD

File: packages/client/wallets/aa/package.json (hunk 3 above)
Comment (alfonso-paella) [reply packages/client/wallets/aa/package.json:3]: i mean in sync w our client UI ones
